hnn_core/tests/conftest.py

```diff
@@ -84,17 +84,17 @@ def _run_hnn_core_fixture(backend=None, n_procs=None, n_jobs=1,
         if reduced:
             params.update({'N_pyr_x': 3,
                            'N_pyr_y': 3,
-                           'tstop': 25,
+                           'tstop': 40,
```

Comment:
this will slow down the tests though. Not sure what's a good solution without bloating this PR further. Maybe an issue to take care of it later? We need to keep track of all the things we plan to do later. I am happy to help in some of these. As long as it's tracked properly, should be fairly quick to take care of

Reply:
It's not too bad in terms of time, just a couple of seconds I'd say in total. It's only used in the severely reduced nets. I like it here because of the test it gave birth to being important in making this PR possible!